sportsdataverse/cfb/cfb_adjusted_epa.py

```
@@ -0,0 +1,368 @@
+        opp = opp.with_columns(
+            pl.col("adjmodelDef").fill_null(fill_strength),
+            pl.col("adjmodelOff").fill_null(fill_strength),
+        )
+    return opp.with_columns(
+        adj_off_epa=pl.col("raw_off_epa") - pl.col("adjmodelDef"),
+        adj_def_epa=pl.col("raw_def_epa") - pl.col("adjmodelOff"),
+    )
+
+
+@overload
+def cfb_adjusted_epa(
```

Flake8 (7.3.0)

[error] multiple statements on one line (def)

(E704)